packages/atomic/src/components/atomic-no-results-v1/atomic-no-results.tsx

```diff
@@ -70,7 +70,7 @@ export class AtomicNoResults {
     const content = this.querySummaryState.hasQuery
       ? this.bindings.i18n.t('no-results-for', {
           interpolation: {escapeValue: false},
-          query: this.wrapHighlight(escape(this.querySummaryState.query)),
+          query: this.wrapHighlight(this.querySummaryState.query),
```

Comment:
Did the atomic-query-summary have that problem?

On another note, if you put a XSS string in the search box, it causes any issues?

Reply:
Perhaps we should also use `dompurify`'s `sanitize()` here, allowing only the `span` tag with the `class` and `part` attributes.